messaging/README.md

```diff
@@ -76,14 +76,14 @@ fetch('https://fcm.googleapis.com/fcm/send', {
 
 ### cURL
 ```
-curl -X POST -H "Authorization: key=YOUR-SERVER-KEY" -H "Content-Type: application/json" -d '{
+curl -X POST -H "Authorization: key=YOUR-ACCESS-TOKEN" -H "Content-Type: application/json" -d '{
   "notification": {
     "title": "Portugal vs. Denmark",
     "body": "5 to 1",
     "icon": "firebase-logo.png",
     "click_action": "http://localhost:8081"
   },
-  "to": "YOUR-IID-TOKEN"
+  "to": "YOUR-FCM-TOKEN"
```

Comment:
is FCM the same as IID?

Reply:
In short: yes.

A bit longer: IID has been deprecated. Going forward we refer to tokens used to target FCM clients as FCM tokens. So yes IID tokens are FCM tokens but so are the newer recommended Installation tokens.